compose/cli-command-compatibility.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Docker Compose command compatibility with docker-compose
+keywords: documentation, docs, docker, compose, containers
+title: Docker Compose command compatibility with docker-compose
+---
+
+In the current state, the compose command in the Docker CLI supports most of the docker-compose commands and flags. It is expected to be a drop in replacement for docker-compose. However, there are a few remaining flags that have not been implemented yet. We are prioritizing the implementation of those based on usage metrics and user feedback.
+
+If you see some compose functionality that is not available in the compose command, please [let us know](https://github.com/docker/compose-cli/issues){:target="_blank" rel="noopener" class="_"} so we can prioritize it.
+
+## Commands or flags not yet implemented
+
+The following sub commands are not implemented nor priroitized today. We may add them to our backlog based on user feedback.
+
+* `compose build --memory`
+* `compose build --no-cache`
+* `compose config --no-interpolate`
+* `compose config --services`
+* `compose config --volumes`
+* `compose config --hash`
+* `compose events`
+* `compose images`
+* `compose port`
+* `compose pull --ignore-pull-failures`
+* `compose push --ignore-push-failures`
+* `compose run --service-ports`
+
+## Flags that will not be implemented
+
+Some flags are already deprecated in docker-compose or are not relevant in the go implementation of the compose command in Docker CLI.
+Here is a list of flags that we don't plan to implement in the compose command:
```

Comment:
Some of the flags below have been deprecated in docker-compose, or are not relevant for the Compose in Docker CLI.  The list below includes the flags that we are not planning to support in Compose in Docker CLI